nimble/controller/src/ble_ll_adv.c

```diff
@@ -891,6 +931,7 @@ ble_ll_adv_tx_done(void *arg)
     ble_ll_state_set(BLE_LL_STATE_STANDBY);
 
     ble_ll_adv_active_chanset_clear(advsm);
+    advsm->flags &= ~BLE_LL_ADV_SM_FLAG_PERIODIC_SYNC_SENDING;
```

Comment:
`#ifdef`

Reply:
let's use `ble_ll_adv_flags_clear()` and `ble_ll_adv_flags_set()`